docs/releases/1.10.txt

```diff
@@ -365,6 +365,16 @@ Models
 
 * A proxy model may now inherit multiple proxy models that share a common
   non-abstract parent class.
+* `auto_now` for
```

Comment:
Going to omit the release notes as these points seem more like bug fixes rather than "Minor features".